wiki/Article_Style_Guide/en.md

```diff
@@ -281,7 +324,7 @@ For example, use:
 - `You will` instead of `You'll`
 - `It is` instead of `It's`
 
-Know that the possessive form of a word (e.g. `the player's` or `the skinner's`) are the expection (as these are not contractions).
+Know that the possessive form of a word (e.g. `the player's` or `the skinner's`) are the expection since these are not contractions.
```

Comment:
Keep consistent with singulars and plurals. Plural feels a bit more fitting here imo.
`... posessive forms of words (...) are exceptions since ...`